src/Debug.mo

```diff
@@ -1,14 +1,32 @@
-/// Debugging aids
+/// Utility functions for debugging.
+///
+/// Import from the base library to use this module.
+/// ```motoko name=import
+/// import Debug "mo:base/Debug";
+/// ```
 
 import Prim "mo:⛔";
 module {
+  /// Prints `text` to output stream.
+  ///
+  /// NOTE: What this output stream is depends on your execution environment.
```

Comment:
```suggestion
  /// NOTE: The output is in the replica log. When running on the main net, it is not visible to the developers.
```

Reply:
Added

Reply:
@chenyan-dfinity is it visible to anyone, or just discarded completely?

Reply:
The current behavior is to discard debugPrint in production, but they can change it. In any case, it's not visible to developers.